Tab.cpp

```diff
@@ -59,9 +69,11 @@ HRESULT Tab::Init(IWebView2Environment* env, bool shouldBeActive)
         // Enable listening for security events to update secure icon
         RETURN_IF_FAILED(m_contentWebView->CallDevToolsProtocolMethod(L"Security.enable", L"{}", nullptr));
 
+        BrowserWindow::CheckFailure(m_contentWebView->GetDevToolsProtocolEventReceiver(L"Security.securityStateChanged", &m_contentReceiver), L"");
```

Comment:
Would be nice to rename m_contentReceiver to something about the CDP event that it receives. This is for the securityStateChanged CDP event, so m_securityStateChangedReceiver or something like that maybe?